app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -69,16 +69,18 @@ def error_messages
         def extra_attributes
           if true_id_product&.dig(:AUTHENTICATION_RESULT).present?
             attrs = response_info.merge(true_id_product[:AUTHENTICATION_RESULT])
-            attrs.reject do |k, _v|
-              PII_EXCLUDES.include? k
+            attrs.reject! do |k, _v|
+              PII_EXCLUDES.include?(k) || k.start_with?('Alert_')
```

Comment:
Is there a test case which covers what we're trying to do with the `Alert_` exclusions?

Reply:
Test added.